samples/apps/copilot-chat-app/webapp/src/libs/services/SpeechService.ts

```diff
@@ -44,39 +38,15 @@ export class SpeechService {
         return new speechSdk.SpeechRecognizer(speechConfig, audioConfig);
     }
 
-    private invokeTokenAsync = async (): Promise<TokenResponse> => {
-        const result = await this.getAzureSpeechTokenAsync<TokenResponse>({
-            commandPath: `speechToken`,
-            method: 'GET',
-        });
-        return result;
-    };
-
-    private readonly getAzureSpeechTokenAsync = async <TokenResponse>(
-        request: SpeechServiceRequest,
-    ): Promise<TokenResponse> => {
-        const { commandPath, method } = request;
-
-        try {
-            const requestUrl = new URL(commandPath, this.serviceUrl);
-            const response = await fetch(requestUrl, {
-                method: method ?? 'GET',
-                headers: { 'Content-Type': 'application/json' },
-            });
-
-            if (!response.ok) {
-                throw Object.assign(new Error(response.statusText + ' => ' + (await response.text())));
-            }
+    private invokeTokenAsync = async (accessToken: string): Promise<TokenResponse> => {
```

Comment:
What does invokeToken mean? Are you initializing a new speech service with the token or just grabbing a new token?  If the latter, let's change this to getTokenAsync